src/main/java/seedu/address/model/event/Event.java

```diff
@@ -0,0 +1,69 @@
+package seedu.address.model.event;
+
+import static seedu.address.commons.util.CollectionUtil.requireAllNonNull;
+
+import java.time.LocalDateTime;
+import java.util.Objects;
+
+import seedu.address.model.person.Name;
+
+public class Event {
+
+    // Identity fields
+    private final Name name;
+
+
+    // Data fields
+    private final LocalDateTime localDateTime;
+    private final String description;
+
+    /**
+     * Constructor for Event.
+     * @param name name of event
+     * @param description description of event
+     * @param localDateTime date and time of event
+     */
+    public Event(Name name, String description, LocalDateTime localDateTime) {
+        requireAllNonNull(name, localDateTime);
+        this.name = name;
```

Comment:
Should we consider including validation on what constitutes a valid `name`? e.g. cannot contain only spaces, cannot be an empty string, etc. May be worth looking at the `Email` class as a good example for validation.